python/graphstorm/trainer/glem_np_trainer.py

```diff
@@ -246,10 +246,10 @@ def _prepare_batch(input_nodes, seeds, blocks, is_labeled=True):
                          input_nodes_u=input_nodes_u)
             profiler.record('train_forward')
 
-            self.optimizer.zero_grad()
+            self.optimizer.zero_grad(optimize_sparse_params=model.module.training_sparse_embed)
```

Comment:
this code may not be able to run in the standalone mode.

Reply:
Will support standalone mode in GLEM trainer in another PR.